README.md

```diff
@@ -21,7 +21,8 @@ you might also find an early build of [python3.10] useful
 
 | Code   | Description                                            |
 |--------|--------------------------------------------------------|
-| YTT101 | `sys.version[:...]` referenced (python3.10)            |
+| YTT101 | `sys.version[:1]` referenced (python3.10)              |
```

Comment:
Is this a `python10` issue?

Reply:
Good call. I can reword `YTT101` to only reference `sys.version[:3]` and introduce `YTT303` for `:1`?

Reply:
Yeah let's do that

Reply:
OK updated